opencti-platform/opencti-graphql/src/utils/template/__executiveSummary.template.ts

```diff
@@ -0,0 +1,158 @@
+const executiveSummaryContent = (containerType: string) => {
+  let typeLabel = 'Types';
+  let typeWidget = '$types';
+  if (containerType === 'Grouping') {
+    typeLabel = 'Context';
+    typeWidget = '$context';
+  }
+  const reliabilityForReport = containerType === 'Report'
+    ? `<tr>
+      <td><strong>Self reliability</strong></td>
+  <td>$reportReliability</td>
+  </tr>`
+    : '';
+  return `
+    <div>
+      <h2>Executive report</h2>
+      
+      <h3>1. Details</h3>
+      <table>
+        <tbody>
+          <tr>
+            <td><strong>Creation date</strong></td>
+            <td>$containerCreationDate</td>
+          </tr>
+          <tr>
+            <td><strong>Last update</strong></td>
```

Comment:
Is it ok that templates texts are not translated?

Reply:
for the moment there are not, translations questions will be answered in the next chunks